From c8f0ac9b429e1ff73a3e87247193c35c66212540 Mon Sep 17 00:00:00 2001
From: Shwetha K Acharya <sacharya@redhat.com>
Date: Tue, 2 Apr 2019 12:06:53 +0530
Subject: [PATCH 41/52] tests/geo-rep: Build failed in Jenkins for test
 bug-1600145.t

Problem: the ((strcmp (key, "cluster.brick-multiplex") == 0))
comparision in cli/src/cli-cmd-parser.c is expecting
either yes or no confirmation from cli, which is not handled
in bug-1600145.t, causing test to wait till timeout and
then fail.

Solution: Passing yes as pipeline to
`gluster v set all cluster.brick-multiplex on` in bug-1600145.t

Label: DOWNSTREAM ONLY

Change-Id: I1a6c2a992b65380cea145fd1c46d22ec1251c77e
Signed-off-by: Shwetha K Acharya <sacharya@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/166694
Reviewed-by: Sunil Kumar Heggodu Gopala Acharya <sheggodu@redhat.com>
Tested-by: Sunil Kumar Heggodu Gopala Acharya <sheggodu@redhat.com>
Reviewed-by: Sunny Kumar <sunkumar@redhat.com>
---
 tests/00-geo-rep/bug-1600145.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/00-geo-rep/bug-1600145.t b/tests/00-geo-rep/bug-1600145.t
index 1d38bf9..359bc4f 100644
--- a/tests/00-geo-rep/bug-1600145.t
+++ b/tests/00-geo-rep/bug-1600145.t
@@ -29,7 +29,7 @@ slave_mnt=$M1
 
 ##create_and_start_master_volume
 TEST $CLI volume create $GMV0 replica 2 $H0:$B0/${GMV0}{1,2};
-gluster v set all cluster.brick-multiplex on
+yes | gluster v set all cluster.brick-multiplex on
 TEST $CLI volume start $GMV0
 
 ##create_and_start_slave_volume
-- 
1.8.3.1

